docs/sources/get-started/quick-start.md

```diff
@@ -194,165 +192,337 @@ You can view your logs using the command line interface, [LogCLI](/docs/loki/<LO
+  metric_aggregation:
+    loki_address: localhost:3100
+
+ruler:
+  enable_alertmanager_discovery: true
+  enable_api: true
+  
+frontend:
+  encoding: protobuf
+
+compactor:
+  working_directory: /tmp/loki/retention
+  delete_request_store: filesystem
+  retention_enabled: true
+```
+To summarize the configuration file:
+* **auth_enabled**: This is set to false, meaning Loki does not need a [tenant ID](https://grafana.com/docs/loki/<LOKI_VERSION>/operations/multi-tenancy/) for ingest or query. Note that this is not recommended for production environments. When deploying the Loki helm chart, this is set to true by default.
+* **server**: Defines the ports Loki listens on, the log level, and the maximum number of concurrent gRPC streams.
+* **common**:  Defines the common configuration for Loki. This includes the instance address, storage configuration, replication factor, and ring configuration.
+* **query_range**: This is configured to tell Loki to use inbuilt caching for query results. In production environments of Loki this is handled by a separate cache service such as memcached.
+* **limits_config**: Defines the global limits for all Loki tenants. This includes enabling specific features such as metric aggregation and structured metadata. Limits can be defined on a per tenant basis, however this is considered an advanced configuration and for most use cases the global limits are sufficient.
+* **schema_config**: Defines the schema configuration for Loki. This includes the schema version, the object store, and the index configuration.
+* **pattern_ingester**: Enables pattern ingesters which are used to discover log patterns. Mostly used by Grafana Logs Drilldown.
+* **ruler**: Enables the ruler component of Loki. This is used to create alerts based on log queries.
+* **frontend**: Defines the encoding format for the frontend. In this case it is set to `protobuf`.
+* **compactor**: Defines the compactor configuration. Used to compact the index and mange chunk retention.
+
+The above configuration file is a basic configuration file for Loki. For more advanced configuration options, refer to the [Loki Configuration](https://grafana.com/docs/loki/<LOKI_VERSION>/configuration/) documentation.
+
+<!-- INTERACTIVE page step7.md END -->
+
+<!-- INTERACTIVE page step8.md START -->
+
+### Grafana Loki Datasource
+
+The final piece of the puzzle is the Grafana Loki datasource. This is used by Grafana to connect to Loki and query the logs. Grafana has multiple ways to define a datasource;
+* **Direct**: This is where you define the datasource in the Grafana UI.
+* **Provisioning**: This is where you define the datasource in a configuration file and have Grafana automatically create the datasource.
+* **API**: This is where you use the Grafana API to create the datasource.
+
+In this case we are using the provisioning method. Instead of mounting the Grafana configuration directory, we have defined the datasource in the `docker-compose.yml` file:
```

Comment:
```suggestion
In this case we are using the provisioning method. Instead of mounting the Grafana configuration directory, we have defined the data source in this portion of the `docker-compose.yml` file:
```